<?xml version="1.0"?>
<launch>
    <!-- <master auto="start"/>  -->
    <!--   <param name="use_sim_time" value="true" /> -->

    <!--     param file -->
    <param name="param_yaml_file" value="$(find patrol_robot)/param/fws_sim.yaml" />

    <!--   robot model -->
    <node pkg="patrol_robot" type="fws_model.py" name="fws_model"/>

    <!--     tf_manager -->
    <node pkg="patrol_robot" type="tf_manager.py" name="tf_manager"/>

    <!--   map_server -->
    <node pkg="map_server" type="map_server" name="map_server" args="$(find patrol_robot)/map/fws_map_sim.yaml"/>

    <!--   if sim, process vlp16 in sim -->
    <include file="$(find patrol_robot)/launch/fws_handle_sim_vlp16.launch" />

    <!--      als -->
    <include file="$(find als_ros)/launch/mcl_patrol_robot.launch" />

    <!--     fws_simple_move_base_server -->
    <node pkg="patrol_robot" type="fws_simple_move_base_server.py" name="fws_simple_move_base_server" output="screen" />

    <!--     fws_simple_move_base_server -->
    <node pkg="patrol_robot" type="simple_move_base_client.py" name="simple_move_base_client" output="screen" />


<!--      -->
<!--      --><!--   task manager, the clients for move base and line track -->
<!--     <node pkg="patrol_robot" type="task_manager.py" name="task_manager" output="screen" /> -->
<!--      -->
<!--      --><!--   line_track server -->
<!--     <node pkg="patrol_robot" type="line_track_server.py" name="line_track_server" output="screen" /> -->
<!--      -->
<!--      --><!--   move_base -->
<!--     <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen"> -->
<!--     <rosparam file="$(find patrol_robot)/param/costmap_common_params.yaml" command="load" ns="global_costmap" /> -->
<!--     <rosparam file="$(find patrol_robot)/param/costmap_common_params.yaml" command="load" ns="local_costmap" /> -->
<!--     <rosparam file="$(find patrol_robot)/param/local_costmap_params.yaml" command="load" /> -->
<!--     <rosparam file="$(find patrol_robot)/param/global_costmap_params.yaml" command="load" /> -->
<!--     <rosparam file="$(find patrol_robot)/param/base_local_planner_params.yaml" command="load" /> -->
<!--     <param name="planner_frequency" value="1" /> -->
<!--     <param name="controller_frequency" value="5" /> -->
<!--     <param name="controller_patience" value="15" /> -->
<!--     <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS" /> -->
<!--     </node> -->
<!--      -->
<!--      --><!--   simple_move_base server -->
<!--     <node pkg="patrol_robot" type="simple_move_base_server.py" name="simple_move_base_server" output="screen" /> -->
<!--      -->
<!--      --><!--   webgui -->
<!--     <node pkg="patrol_robot" type="webgui.py" name="webgui" output="screen" /> -->

    <!-- rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find patrol_robot)/rviz_cfg/fws_sim.rviz"/>
</launch> 

